web/views_whiteboard.py

```
@@ -0,0 +1,212 @@
+from django.shortcuts import render, get_object_or_404, redirect
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)